src/sentry/api/endpoints/organization_measurements_meta.py

```diff
@@ -15,7 +15,7 @@
 @region_silo_endpoint
 class OrganizationMeasurementsMeta(OrganizationEventsEndpointBase):
     publish_status = {
-        "GET": ApiPublishStatus.UNKNOWN,
+        "GET": ApiPublishStatus.EXPERIMENTAL,
```

Comment:
custom measurements is a little up in the air, so marking it as experimental too